gent_base.theme

```diff
@@ -292,11 +361,29 @@ function gent_base_preprocess_element_information(array &$variables) {
  */
 function gent_base_preprocess_form_element(&$variables) {
   $element = &$variables['element'];
-  if (!isset($element['#help'])) {
-    return;
+  if (isset($element['#help'])) {
+    $element['#information'] = $element['#help'];
   }
 
-  $element['#information'] = $element['#help'];
+  if (isset($element['#description'])) {
+    $element['#information'] = $element['#description'];
+  }
+
+  if (isset($element['#in_fieldset'])) {
+    $variables['in_fieldset'] = $element['#in_fieldset'];
```

Comment:
Variables are only set when the element has an in_fieldset.
Better to set a value anyway?

```php
$variables['in_fieldset'] = $element['#in_fieldset'] ?? NULL;
$variables['parent_type'] = $element['#parent_type'] ?? NULL;
$variables['stacked'] = $element['#stacked'] ?? NULL;
$variables['has_row'] = $element['#has_row'] ?? NULL;
```